03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/11 CBOT/02 pre-market-hours.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
The name of this file should be `03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/11 CBOT/02 Pre-Market Hours.php`, not `03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/11 CBOT/02 pre-market-hours.php`. Looks like some of the other files have been renamed too